sdk/cloudmachine/Azure.Projects.Provisioning/src/Features/OfxProjectFeature.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Projects.Core;
+using Azure.Projects.Storage;
+
+namespace Azure.Projects.Ofx;
+
+public class OfxProjectFeature : AzureProjectFeature
+{
+    private readonly BlobContainerFeature _blobContainer;
+
+    public OfxProjectFeature()
+    {
+        _blobContainer = new BlobContainerFeature("default", isObservable: true);
```

Comment:
Why is blob the only implicit feature added?

Reply:
isObservable: true add all the rest. I am assuming we will have more as we add more OFX APIs)